operator/apis/execution/v1/scancompletionhook.go

```diff
@@ -39,13 +39,26 @@ type ScanCompletionHookSpec struct {
 	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
 	// Important: Run "make" to regenerate code after modifying this file
 
+	// Defines weather the hook should be able to change the findings or is run in a read only mode.
+	Type HookType `json:"type"`
+
 	// Image is the container image for the hooks kubernetes job
-	Image            string                        `json:"image,omitempty"`
+	Image string `json:"image,omitempty"`
+	// ImagePullSecrets used to access private hooks images
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
-	Env              []corev1.EnvVar               `json:"env,omitempty"`
-	Type             HookType                      `json:"type"`
+	// Image pull policy. One of Always, Never, IfNotPresent. Defaults to Always if :latest tag is specified, or IfNotPresent otherwise. Cannot be updated. More info: https://kubernetes.io/docs/concepts/containers/images#updating-images
```

Comment:
What is meant here by "Cannot be updated."? @rseedorff 

A update to this value should only affect hooks started after the change, which should be ok, right?

Reply:
Good point - must be a copy/paste issue. This sentence can be deleted.